cpp/src/arrow/compute/exec/test_util.h

```diff
@@ -145,5 +145,38 @@ class Random64Bit {
   std::uniform_int_distribution<uint64_t> dist_;
 };
 
+/*
```

Comment:
Can you use the `/// \brief`... style of commenting found elsewhere in the code base?